### ISSUE TITLE
Pagination state is not maintained correctly

### ISSUE THREAD

**Issue by nisarg73** [IMGIITRoorkee/omniport-frontend-noticeboard#10]
![Hnet-image](https://user-images.githubusercontent.com/25002246/98484026-afea2900-2232-11eb-8c38-b2213b9bc36e.gif)

After opening any notice on `nth` page, upon going back, pagination has `1` as activePage instead of `n`.

**Comment by carpecodeum** [comment]
> ![Hnet-image](https://user-images.githubusercontent.com/25002246/98484026-afea2900-2232-11eb-8c38-b2213b9bc36e.gif)
> 
> After opening any notice on `nth` page, upon going back, pagination has `1` as activePage instead of `n`.

Yes, thanks Nisarg I have to tell you though this issue had been brought to notice before and as this will require some major changes which are already happening as a new noticeboard is already under development so it will be fixed in that.
